notifications/README.md

```diff
@@ -0,0 +1,175 @@
+# Notification
+
+In case we would like to to send notifications using multiple channels like Email
+and HipChat right after some business action (send feedback for example).
+
+The entry point is `SendFeedback` organizer which will organize interactors for:
+create feedback, send notification via Email, and send notification via HipChat.
+
+All notification will be pushed in the background using `ActiveJob`.
+
+```ruby
+# app/interactors/send_feedback.rb
+class SendFeedback
+  include Interactor::Organizer
+
+  organize CreateFeedback, NotifyUserViaEmail, NotifyUserViaHipchat
+end
+
+# app/interactors/notify_user_via_email.rb
+class NotifyUserViaEmail
+  include Interactor
+
+  def call
+    ApplicationMailer.feedback_notification(email, message.content_for(:email)).deliver_later
+  end
+
+  private
+
+  def email
+    context.feedback.employee.email
```

Comment:
@ArturMinnullin check this out https://github.com/fs/rails-base-examples/pull/11/commits/603cecd75abeb3f16ead0fbd5a449fd11f8d45a4